models/stg_recharge__discount.sql

```diff
@@ -38,8 +38,8 @@ final as (
         duration,
         duration_usage_limit,
         minimum_order_amount,
-        once_per_customer,
-        applies_once,
+        once_per_customer, 
+        applies_once, 
```

Comment:
This both look like they are booleans. If so, we should rename to be consistent with our boolean naming conventions. Although, I am having a hard time turning these into `is_applies_once` 🤔. What are your thoughts?

Reply:
I took a closer look, and it looks like `applies_once` is only in our sample dataset and not in the ERD, so I will just remove it. That leaves `is_once_per_customer`, which sounds ok to me.

Reply:
Great! Thanks for confirming and making the appropriate updates.